website/content/commands/connect/ca.mdx

```diff
@@ -89,11 +89,13 @@ Usage: `consul connect ca set-config [options]`
 
 Corresponding HTTP API Endpoint: [\[PUT\] /v1/connect/ca/configuration](/api-docs/connect/ca#update-ca-configuration)
 
-#### API Options
+The output looks like this:
```

Comment:
Moved the output to just below the usage (rather than after the long list of general API options).